src/css/styles.css

```diff
@@ -9,13 +9,11 @@
   overflow: hidden;
   width: 80px;
   height: 80px;
-  background: linear-gradient(to bottom, white 0%,#744662 75%,#444444 76%,#444444 100%);
   background-color: #dddddd !important;
 }
 
 .dropdown-menu {
   width: 266px;
-  background: linear-gradient(#744662, #dddddd);
   background-color: #ababab !important;
```

Comment:
I would suggest to use SASS. It is always better to assign colors to variables in order to make css files cleaner and more clear.